cantor-phoenix/src/main/java/com/salesforce/cantor/phoenix/EventsOnPhoenix.java

```diff
@@ -0,0 +1,492 @@
+package com.salesforce.cantor.phoenix;
+
+import com.salesforce.cantor.Events;
+import com.salesforce.cantor.jdbc.AbstractBaseEventsOnJdbc;
+
+import javax.sql.DataSource;
+import java.io.IOException;
+import java.sql.*;
+import java.util.*;
+
+import org.apache.commons.collections4.map.MultiKeyMap;
+
+public class EventsOnPhoenix extends AbstractBaseEventsOnJdbc implements Events {
+
+    //TODO: in the future create tables and sequences in one batch
```

Comment:
Let's do the TODO - we can either have a long string here, or put the sql statement in a file in /resources and read from that.